docs/book/src/tasks/experimental-features/cluster-class/operate-cluster.md

```diff
@@ -42,3 +52,96 @@ kubeadmcontrolplane.controlplane.cluster.x-k8s.io/clusterclass-quickstart-XXXX
 NAME                                                                             CLUSTER                   REPLICAS   READY   UPDATED   UNAVAILABLE   PHASE     AGE     VERSION
 machinedeployment.cluster.x-k8s.io/clusterclass-quickstart-linux-workers-XXXX    clusterclass-quickstart   1          1       1         0             Running   7m29s   v1.22.0
 ```
+
+## Scale a MachineDeployment
```

Comment:
Let make a short intro before getting into the example
```suggestion
## Scale a MachineDeployment
The `spec.topology` field can be used as a single point of control for all the MachineDeployment in your Cluster.
```
Same below

Reply:
Also somewhere (at the end of the paragraph or in a dedicated paragraph) we should list the other changes we can do to an existing MD